### DISCUSSION TITLE
How to solve: Selected row changes bgColor to a "transparent" one, causing overlapping data visualization on the pinned columns

### DISCUSSION THREAD

**Discussion by ingegneri-bruno-clorox** [KevinVandy/material-react-table#1036] [Q&A] [answered]
Hi. I'm facing the following issue:
![Capture1](https://github.com/KevinVandy/material-react-table/assets/116366661/5833e908-26f4-4b54-925a-82881dc9692b)

I got a table with **pinned columns to the left and to the right**. I'm also with the **select row functionality enabled**.
But, whenever I select a row, it changes the background color to a partially transparent one, causing this **overlapping data visualization on the pinned columns** _(as on the first row of the screenshot above)_.

How can I fix this issue, keeping all functionalities?

**Comment by KevinVandy** [comment, marked as answer]
What version of MRT? This was fixed in the latest versions of MRT v2

<img width="733" alt="image" src="https://github.com/KevinVandy/material-react-table/assets/28243511/7b9f41b7-fe24-4824-9e08-f11441096604">

**Reply by ingegneri-bruno-clorox** [reply]
I was using **2.0.5** and saw that it's mentioned as fixed on **2.9.0**

> Fixed row hover opacity style issues for pinned columns and selected rows with ::before and ::after pseudo elements in pinned cells

I update and it solved the issue. Thanks!
